### ISSUE TITLE
Progress bar always disappears right after pressing the "generate" button 

### ISSUE THREAD

**Issue by RaphaelMaschinsen** [TheLastBen/fast-stable-diffusion#1943]
I'm on windows/chrome and use the stable diffusion colab notebook. I have also tried Ngrok and the same thing happens (it is more stable with Ngrok ). When I start a big batch there is no way to interrupt it other than force quitting in the colab notebook itself which is tedious. Is anyone else experiencing the same problem?

**Comment by TheLastBen** [comment]
it's colab problem not specifically to the notebook, check the box "use_latest_working_commit"

**Comment by Omenizer** [comment]
> I'm on windows/chrome and use the stable diffusion colab notebook. I have also tried Ngrok and the same thing happens (it is more stable with Ngrok ). When I start a big batch there is no way to interrupt it other than force quitting in the colab notebook itself which is tedious. Is anyone else experiencing the same problem?

Yes, the interface is extremely unresponsive lately, sometimes it's nearly impossible to work and I have to reload the page after every gen. Progress bar almost always disappears immediately, and images take a minute to show up after a 20 second batch time...

**Comment by TheLastBen** [comment]
it's less problematic when using ngrok

**Comment by modulator** [comment]
> > I'm on windows/chrome and use the stable diffusion colab notebook. I have also tried Ngrok and the same thing happens (it is more stable with Ngrok ). When I start a big batch there is no way to interrupt it other than force quitting in the colab notebook itself which is tedious. Is anyone else experiencing the same problem?
> 
> Yes, the interface is extremely unresponsive lately, sometimes it's nearly impossible to work and I have to reload the page after every gen. Progress bar almost always disappears immediately, and images take a minute to show up after a 20 second batch time...

I have this same problem for - and I'm using MacOS/Chrome. 

This has been happening for several weeks now. The status bar disappears immediately after clicking "Generate" button, and it the "Generate" immediately resets to default state. with there's no obvious way to stop generating.  For the first few weeks, I was able to kill the Generation by switching to Img2Img tab, clicking THAT "Generate" button, then clicking the "Interrupt/Skip" buttons repeatedly until it stopped. But lately even THAT is not working...

ALSO - Not sure if it's the same or different issue - but lately, when I set ANY multiple Batch Count (for example, 100), I usually only get ONE (or at most, some random low quantity) before Generation stops.

That said, I LOVE these tools. Keep up the great work!

**Comment by TheLastBen** [comment]
check the box "latest_working_commit", it should help with the progress bar
